src/cpp/src/image_generation/numpy_utils.hpp

```diff
@@ -51,6 +51,10 @@ ov::Tensor concat(ov::Tensor tensor_1, ov::Tensor tensor_2, int axis);
 
 void batch_copy(ov::Tensor src, ov::Tensor dst, size_t src_batch, size_t dst_batch, size_t batch_size = 1);
 ov::Tensor repeat(const ov::Tensor input, const size_t num_images_per_prompt);
+ov::Tensor squeeze(const ov::Tensor& tensor, size_t dim);
+ov::Tensor unsqueeze(const ov::Tensor& tensor, size_t dim);
+std::vector<ov::Tensor> split(const ov::Tensor& tensor);
+ov::Tensor stack(const std::vector<ov::Tensor>& tensors);
```

Comment:
Great to provide these utils function, 
but I still suggest to move them to modules_genai's utils, because I will reduce rebase conflict when we rebase codes in the future. 
What do you think？